### DISCUSSION TITLE
Persisted docker containers for "NV_CHAD"

### DISCUSSION THREAD

**Discussion by P-Y-R-O-B-O-T** [NvChad/nvchad.github.io#209] [Ideas]
I have an idea to implement persisted docker containers for NV_CHAD.

In the website docs there is an option to install and test NV_CHAD config in a docker container. I would like to add an enhanced option of that alongside it where user can edit and store all of the programming projects in the volumes attached to docker container. This will also allow to tweak NV_CHAD inside the container itself, and if it gets messed up, user just need to recreate the container instead of going through full installation.

User will also be able to use TMUX and NV_CHAD at the same time in a single container with all the additional tools required along with the project. Users will also be able to install new tools as well and manage different versions of them.
Also there will be a reduction to download the nerd fonts externally if we were working on a local system.

I wanna update/add the docs for this in the website docs.

Thank you
Hoping for positive a positive reply

**Comment by siduck** [comment]
@P-Y-R-O-B-O-T  https://github.com/NvChad/nvchad.github.io/blob/main/src/components/docpage/install.tsx#L38

you can add the docker stuff but not tmux as not all docker users use tmux

**Reply by P-Y-R-O-B-O-T** [reply]
ok

Firstly I will create and test the thing then update, might take a day or two

Thanks for taking up the contribution

**Comment by P-Y-R-O-B-O-T** [comment]
Hi, everything is complete from my side (creation, testing, updating the webdocs).

I have created a pull request, kindly verify the same and merge

https://github.com/NvChad/nvchad.github.io/pull/210

Hoping all goes well ...

If there are any issues; let me know

**Comment by P-Y-R-O-B-O-T** [comment]
Hey, I understand the point, but it will be good to have a reference to that thing also.

I also understand the problem, you can create your own repo in NvChad account, I will push changes to that repo.
